digitalocean/firewalls.go

```diff
@@ -0,0 +1,322 @@
+package digitalocean
+
+import (
+	"github.com/digitalocean/godo"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func firewallSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"firewall_id": {
```

Comment:
If we make the other changes I suggestion, this can be removed from the share schema.